src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -2,15 +2,17 @@
 {
     using System;
     using System.Collections.Generic;
+    using System.Diagnostics;
     using System.Threading;
     using System.Threading.Tasks;
 
     static class TaskExtensions
     {
+        //this method will not timeout a task if the debugger is attached.
```

Comment:
nitpicking, but could this be `// this method will not timeout if the debugger is attached.`